packages/ui/src/components/address.tsx

```diff
@@ -46,18 +46,16 @@ Address.Country = function Country(props: SelectProps) {
   return (
     <Select
       defaultValue={country}
-      onChange={(e) => {
-        setCountry(e.currentTarget.value as SupportedCountry);
+      onChange={(event) => {
```

Comment:
Let's keep using `e` whenever we're referring to an `event`, no need to rename. Apply everywhere please!